src/scripts/main.js

```diff
@@ -6,9 +6,14 @@
  */
 
 import Scroll from "./smoothScroll/scrollModule.js";
+import ScrollReveal from "./scrollReveal/scrollRevealModule.js";
+import ToogleSwitch from "./toogleSwitch/toogleSwitch.js";
+
 const scroll = new Scroll({ class: ".grav-c-cta" });
 scroll.init();
 
-import ScrollReveal from "./scrollReveal/scrollRevealModule.js";
 const scrollReveal = new ScrollReveal();
 scrollReveal.init();
+
+const toogleSwitch = new ToogleSwitch("[type=button][aria-pressed]");
```

Comment:
I think we can hard code the `[type=button][aria-pressed]` inside the class. Can't think of any other elements you'd need to apply that specific event handler to (certainly, we don't have any right now).